### PR TITLE
handle lti3 controller case

### PR DESCRIPTION
**Pull request by smith750**
Handle wildcard extraction in lti3 controller in addition to oidc controller.

### CODE REVIEW

File: src/main/java/net/unicon/lti/controller/lti/LTI3Controller.java [hunk 1]
```diff
@@ -100,7 +100,12 @@ public String lti3(HttpServletRequest req, HttpServletResponse res, Model model)
         String link = req.getParameter("link");
         String altDomain = req.getHeader("x-formated-host");
```
Comment (HotchkissLuLe) [inline src/main/java/net/unicon/lti/controller/lti/LTI3Controller.java:101]: I know very little about this, but is this a typo? Formatted is spelled with 2 t's

File: src/main/java/net/unicon/lti/controller/lti/LTI3Controller.java [hunk 2]
```diff
@@ -100,7 +100,12 @@ public String lti3(HttpServletRequest req, HttpServletResponse res, Model model)
...
         if (StringUtils.isNotBlank(altDomain)){
-            altDomain = DomainUtils.extractDomain(altDomain);
+            String extractedDomain = DomainUtils.extractDomain(altDomain);
+            if (extractedDomain == null && DomainUtils.isWildcardDomain(altDomain, ltiDataService.getLocalUrl())) {
```
Comment (smith750) [inline src/main/java/net/unicon/lti/controller/lti/LTI3Controller.java:104]: @ddelblanco Is altDomain what you would expect here?

Comment (ddelblanco) [reply src/main/java/net/unicon/lti/controller/lti/LTI3Controller.java:104]: now, with the last changes,  it looks good to me

**Review by ddelblanco — APPROVED** [review]

**Review by leannethng — APPROVED** [review]
This looks good to me!

File: src/main/java/net/unicon/lti/controller/lti/LTI3Controller.java (hunk 2 above)
Comment (HotchkissLuLe) [reply src/main/java/net/unicon/lti/controller/lti/LTI3Controller.java:104]: I just want to confirm that we do not need to worry about checking the domain.url here?

**Review by HotchkissLuLe — APPROVED** [review]
Pending confirmation on the comment, I think this is exactly what we need :)

**Pull request merged**